packages/runtime/src/useCases/common/RuntimeErrors.ts

```diff
@@ -134,7 +134,7 @@ class Notifications {
     }
 
     public cannotSaveSentNotificationFromPeerMessage(): ApplicationError {
-        return new ApplicationError("error.runtime.notifications.cannotSaveSendNotificationFromPeerMessage", "Cannot send Notification from peer message.");
+        return new ApplicationError("error.runtime.notifications.cannotSaveSentNotificationFromPeerMessage", "Cannot send Notification from peer message.");
```

Comment:
The error message seems misleading to me. If I understand it correctly, the use case SentNotification is meant to save a LocalNotification for a Notification that was sent inside a Message by oneself. If the owner of the Message is not the Identity itself, this error is thrown. In this case, I'd expect an error message like `The Message ${messageId} was received from a peer, but an own Message is expected here to save its Notification content.`.

Besides, also the naming of the SentNotificationUseCase seems not ideal to me, as it doesn't appropriately describe what happens inside of it. However, this is probably outside the scope of this PR.